code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -108,10 +111,12 @@
 //Slime people are able to split like slimes, retaining a single mind that can swap between bodies at will, even after death.
 
 /datum/species/jelly/slime
-	name = "Slimeperson"
+	name = "Xenobiological Slimeperson"
 	id = "slime"
 	default_color = "00FFFF"
 	species_traits = list(SPECIES_ORGANIC,MUTCOLORS,EYECOLOR,HAIR,FACEHAIR,NOBLOOD,TOXINLOVER)
+	mutant_bodyparts = list("mam_tail", "mam_ears", "taur")

```

Comment:
add //CIT CHANGE

Reply:
Are you wanting //CIT CHANGE added to the end of it?